generate_version_translation_unit.sh

```diff
@@ -13,9 +13,17 @@ char const Version[] =
 }  // namespace base
 }  // namespace principia"
 
-echo -e "$VERSION_TEMPLATE" | 
-sed "s/%%DATE%%/`date -d $(git log -1 --format=%cd --date=iso-strict) -u +%Y-%m-%dT%H:%M:%SZ`/" |
-sed "s/%%VERSION%%/`git describe --tags --always --dirty --abbrev=40 --long`/" > $TEMPORARY_FILE
+PLATFORM=$(uname -s)
+if [ "$PLATFORM" == "Darwin" ]; then
+  echo -e "$VERSION_TEMPLATE" |
+  sed "s/%%DATE%%/`date -j -u -f \"%Y-%m-%d %H:%M:%S %z\" \"$(git log -1 --format=%cd --date=iso)\"`/" |
```

Comment:
So that string is what parses the `git log` output and its right because `--date=iso` gives that.  The problem is that it needs the proper format string for output, which I appear to have dropped